repos/fdbt-site/src/pages/products/exports.tsx

```diff
@@ -222,10 +226,19 @@ export const getServerSideProps = async (ctx: NextPageContextWithSession): Promi
     const noc = getAndValidateNoc(ctx);
 
     const operatorHasProducts = (await getAllProductsByNoc(noc)).length > 0;
+    const selectExportAttribute = getSessionAttribute(ctx.req, SELECT_EXPORTS_ATTRIBUTE);
+    const currTime = new Date().getTime() / 1000;
+
+    const exportStarted = !!selectExportAttribute && currTime - Number(selectExportAttribute.exportStarted) > 5;
+
```

Comment:
I think we should move this logic into a function called exportHasStarted, and then we can check it via a unit test. And it means we can use the function in multiple places.

Reply:
was also thinking the same and it can be called at multiple places